doc/source/ray-observability/ray-logging.rst

```diff
@@ -4,6 +4,38 @@ Logging
 =======
 This document will explain Ray's logging system and its best practices.
 
+Internal Ray Logging Configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+When ``import ray`` is executed, Ray's logging system is initialized, generating a sensible configuration given in ``python/ray/log.py``. The default logging level is ``logging.INFO``.
+
+Ray uses `rich <https://github.com/Textualize/rich>`__ to generate log messages that are well formatted and easily identified. If ``rich`` is not installed, a simple fallback formatter is used. Rich log messages display contextual information including the Ray component from which the message was generated as well as the filename and line number which are not available with the simple fallback logger.
+
+Changing Ray's Internal Logging Configuration
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Modifying Ray's logging configuration can be done by changing the configuration in ``python/ray/log.py`` according to `the python logging documentation <https://docs.python.org/3/howto/logging.html>`__. Ray uses a `dictConfig <https://docs.python.org/3/library/logging.config.html#logging.config.dictConfig>`__ to specify logging configuration.
+
+Logging configuration should be kept inside ``python/ray/log.py`` to avoid spreading conflicting logging configurations across the database. Generally speaking there is no need to call ``logging.basicConfig``, as logging configuration is done at import time.
```

Comment:
```suggestion
Logging configuration should be kept inside ``python/ray/log.py`` to avoid spreading conflicting logging configurations across the database. You do not need to call ``logging.basicConfig`` because logging configuration is done at import time.
```